File: Bus-station-react-frontend/src/dtos/InterCityBusRideDTO.ts
export class InterCityBusRideDTO {
    id: number;
    estimatedTravelTime: number;
    busCompanyName: String;
    timeOfDeparture:String;
    destination:String;
    frequency:String;
    platform:number;

    constructor(){
        this.id = 0,
        this.estimatedTravelTime = 0,
        this.busCompanyName = "",
        this.timeOfDeparture = "",
        this.destination = "",
        this.frequency = "",
        this.platform = 0
    }
}

File: bus_station/src/main/java/com/thesis/bus_station/dto/InterCityBusRideDTO.java
package com.thesis.bus_station.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class InterCityBusRideDTO {

    public Long id;
    public int estimatedTravelTime;
    public String busCompanyName;
    public LocalDateTime timeOfDeparture;
    public String destination;
    public String frequency;
    public int platform;
}
